exponent-view-template/ios/exponent-view-template/AppDelegate.m

```diff
@@ -55,4 +55,9 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
     [[ExponentViewManager sharedInstance] application:application didReceiveRemoteNotification:notification];
 }
 
+- (void)application:(UIApplication *)application didRegisterUserNotificationSettings:(UIUserNotificationSettings *)notificationSettings
+{
+    [[ExponentViewManager sharedInstance] application:application didRegisterUserNotificationSettings:notificationSettings];
+}
+
```

Comment:
This is a work in progress so don't worry about breaking/testing it right now. We are adding the ability to semi-`eject` to native code, so `ExponentViewManager` is an API that can work with any `AppDelegate`, not just Exponent's.